lib/ansible/modules/cloud/docker/docker_plugin.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/python
+# coding: utf-8
+#
+# Copyright: (c) 2019, Vladimir Porshkevich (@porshkevich) <neosonic@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = u'''
+module: docker_plugin
+version_added: "2.8"
+short_description: Manage Docker plugins
+description:
+  - Install/remove Docker plugins.
+  - Performs largely the same function as the "docker plugin" CLI subcommand.
+options:
+  name:
+    description:
+      - Name of the plugin to operate on.
+    required: true
+    type: str
+
+  alias:
+    description:
+      - Alias of the plugin to operate on. Same plugin can be installed with different alias.
+    type: str
+
+  plugin_options:
+    description:
+      - Dictionary of plugin settings.
+    type: dict
+
+  state:
+    description:
+      - C(absent) remove the plugin.
+      - C(present) install the plugin, if it does not already exist.
+      - C(enable) enable the plugin.
+      - C(disable) disable the plugin.
+    default: present
+    choices:
+      - absent
+      - present
+      - enable
```

Comment:
```suggestion
      - enabled
      - disabled
```
Otherwise it is an action, not a state.